SDK/bundles/com.ashling.riscfree.debug.opxd.registers/src/com/ashling/riscfree/debug/opxd/registers/core/RiscFreeRegister.java

```diff
@@ -632,4 +637,131 @@ private ILaunchConfiguration getLaunchConfig() {
 		ILaunchConfiguration config = launch.getLaunchConfiguration();
 		return config;
 	}
+
+	public String getDefaultRegisterFilePath() {
+		return rootRegisterFilePath;
+	}
+
+	public String getFormattedRegisterFilePath() {
```

Comment:
It's not just getting formatted register file path, it's creating the file too, so rename the function accordingly.

Reply:
function name changed